templates/windows10.tpl.yaml

```diff
@@ -124,7 +124,7 @@ objects:
                 bus: usb
                 name: tablet
 {% endif %}
-        terminationGracePeriodSeconds: 0
+        terminationGracePeriodSeconds: 30
```

Comment:
I don't think that 30 seconds is a good default for windows. @ksimon1 what do you think? I would rather think into the direction of 1 hour or so.

Reply:
I think you are right. Shutting down windows can take much longer due to e.g. updates.